package.json

```diff
@@ -68,7 +68,7 @@
     "spec": "test/**/*test.js"
   },
   "volta": {
-    "node": "13.13.0",
-    "yarn": "1.22.0"
+    "node": "14.3.0",
```

Comment:
Mostly did this because 14 will eventually be an LTS (in October 2020) so it seems a better target than 13.x.

Reply:
whats up with CI?